generators/python/fastapi/versions.yml

```diff
@@ -1,4 +1,16 @@
 # For unreleased changes, use unreleased.yml
+- version: 1.5.0-rc4
+  irVersion: 53
+  changelogEntry:
+    - type: fix
+      summary: |
+        Use of `IS_PYDANTIC_V2` flag is now only used within universal base models (`pydantic_utilities.py`) and if pydantic version compatibility is not specified
+        or set to `Both`. Concretely, this means that if your configuration specifies a version for Pydantic, then your generated code will respect that version, and
+        generate the code for that specific version of Pydantic, as opposed to always generating code compatible with both versions of Pydantic.
+      fixes:
+        - |
+          `exclude_unset` is now respected when passed in as an option to a model's `.dict` call.
```

Comment:
do we need to release new versions of pydantic and sdk as well?

Reply:
i already asked this; but based on the code it looks like we would not send explicit Nones for all SDKs ?